config/routes.rb

```diff
@@ -8,7 +8,14 @@
 
   get 'account' => 'users#show', as: 'user_root'
 
-  namespace :admin do
-    resources :authentication_methods
+  namespace :api, defaults: { format: 'json' } do
```

Comment:
why did we remove admin routes?

I don't think we should be removing any routes..